src/saving/NewSaveMenu.cs

```diff
@@ -71,7 +76,7 @@ private void ShowOverwriteConfirm(string name)
     {
         // The chosen filename ({0}) already exists. Overwrite?
         overwriteConfirm.DialogText = string.Format(CultureInfo.CurrentCulture,
-            TranslationServer.Translate("CHOSEN_FILENAME_ALREADY_EXISTS"),
+            TranslationServer.Translate("THE_CHOSEN_FILENAME_ALREADY_EXISTS"),
```

Comment:
Seems like an unnecessary change.

Reply:
I don't know what happened to the exactly. Merge commit did this. Maybe accidentally